### ISSUE TITLE
System already had the same command?

### ISSUE THREAD

**Issue by siygle** [mattvr/ShellGPT#16]
Sorry for asking. 
I'm using Mac and seem it already have the same "gpt" command in the system: 

>      gpt – GUID partition table maintenance utility

Is that possible to allow users can rename the default command name?

**Comment by felipecrs** [comment]
You can choose your own name this way:

```console
deno install --reload --allow-all https://deno.land/x/shellgpt/mod.ts --name othergpt
```

**Comment by siygle** [comment]
Thanks!

**Comment by felipecrs** [comment]
lol, it was just a tip from another user. Maybe @mattvr can comment on whether changing the default name is a good idea or not.

**Comment by mattvr** [comment]
@siygle Yeah, there are two options:

1. The included install script should let you select what you want the command to be called. You could make it `ai`, `chat`, etc. – I guess that might not have worked or been missed?
2. If you make sure deno's binary comes before your existing `$PATH`, i.e. `export PATH="$HOME/.deno/bin:$PATH"`, it should also avoid the issue.

**Comment by siygle** [comment]
@mattvr Thanks for the reply. (Yes, I did miss the message during the install script, really sorry about that). 

However, I just found another strange situation. After I installed the ShellGpt,  it kept displaying the upgrade message as follows:

<img width="1004" alt="image" src="https://github.com/mattvr/ShellGPT/assets/173408/f6ae45af-55af-4a39-baf3-fdefb2869c9a">

But even if I run the command in the message
> deno run -A https://deno.land/x/shellgpt/install.ts

This message keeps jumping out. (I really want to upgrade to 0.3.6 to test the new feature `OPENAI_CHAT_URL` for the local model). Is that possible to come from the different VERSION at [here](https://github.com/mattvr/ShellGPT/blob/0.3.6/lib/data.ts#L4)?
